tests/acceptance/collection-navigation-test.js

```diff
@@ -0,0 +1,42 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import freestyleGuide from '../pages/freestyle-guide';
+
+let fooCollection;
+
+let variantListItems = {
+  normal:   ['Normal', '', '', '', '', ''],
+  special:  ['', 'Special', '', '', '', ''],
+  hyper:    ['', '', 'Hyper', '', '', ''  ],
+  classic:  ['', '', '', 'Classic', '', ''],
+  elegant:  ['', '', '', '', 'Elegant', '' ],
+  tasteful: ['', '', '', '', '', 'Tasteful']
+};
+
+moduleForAcceptance('Acceptance | collection navigation', {
+  beforeEach() {
+    freestyleGuide.visit();
+    andThen(() => {
+      fooCollection = freestyleGuide.content.sections(0).subsections(0).collections(0);
+    });
+  }
+});
+
+test('verifying variantListItem selection', (assert) => {
```

Comment:
It took me a little while to unwind this test. I like that it exercises navigation to each variant in a collection. I think we can simplify the approach.

```javascript
let variantKeys = ['normal', 'special', 'hyper', 'classic', 'elegant', 'tasteful'];

// PSUEDOCODE...
// variantKeys.forEach((key, index) => { ...
//   1. selectVariant(key)
//   2. assert title is present... LINE 34 but with assert.equal rather than assert.ok
//   3. assert other variants are hidden... variantKeys.reject((k) =>  k === 'key').forEach((hiddenVariantKey) =>
```

ish... something along those lines. Thoughts @migbar?